packages/picasso.js/src/core/chart-components/ref-line/lines-and-labels.js

```diff
@@ -61,13 +76,15 @@ export function alignmentToNumber(align) {
  * @param {object[]} items - Array of all items (for collision detection)
  * @ignore
  */
-export function createLineWithLabel({ chart, blueprint, renderer, p, settings, items }) {
+export function createLineWithLabel({ chart, blueprint, renderer, p, settings, items, slopeLine }) {
```

Comment:
p should have some more properties to cover for slope line case and we don't need to pass slopeLine here.